Description: Do not build examples
 The examples shouldn't be built as part of the package building process, as
 they are meant to be a documentation aid and are installed as such.
 Without this patch, we would be installing built binaries in the package's
 documentation directory. The same applies for benchmarks.
Author: Andreas Hasenack <andreas@canonical.com>
Forwarded: No
Last-Update: 2018-04-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/Makefile
+++ b/src/Makefile
@@ -38,13 +38,13 @@
 
 TARGETS = libpmem libvmem libpmemblk libpmemlog libpmemobj libpmempool\
 		  libpmemcto libvmmalloc tools
-ALL_TARGETS = $(TARGETS) common librpmem examples benchmarks
+ALL_TARGETS = $(TARGETS) common librpmem
 
 SCOPE_DIRS = $(TARGETS) common librpmem rpmem_common
 
 DEBUG_RELEASE_TARGETS = common libpmem libvmem libpmemblk libpmemlog libpmemobj\
 	libpmempool libvmmalloc librpmem libpmemcto
-RELEASE_TARGETS = tools examples benchmarks
+RELEASE_TARGETS = tools
 
 CLEAN_NO_JE_TARGETS = $(ALL_TARGETS) rpmem_common test
 CLEAN_TARGETS = $(CLEAN_NO_JE_TARGETS) jemalloc
--- a/src/test/Makefile
+++ b/src/test/Makefile
@@ -369,7 +369,6 @@
 	$(VMEM_TESTS)\
 	$(VMMALLOC_DUMMY_FUNCS_TESTS)\
 	$(VMMALLOC_TESTS)\
-	$(EXAMPLES_TESTS)\
 	$(LIBPMEMPOOL_TESTS)\
 	$(LIBPMEMPOOL_MOD_TESTS)\
 	$(DAXIO_TESTS)
